commit 35360f9aeff361126cfac5b398a5a1ec040bb32d
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Tue Sep 5 10:16:07 2017 +0100

    vcsm: Replace SM_STATE_T with sm_state_t
    
    Follow the Linux kernel coding style
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/char/broadcom/vc_sm/vmcs_sm.c b/drivers/char/broadcom/vc_sm/vmcs_sm.c
index a27a5e9d4cb9..56ab1c8fd6f5 100644
--- a/drivers/char/broadcom/vc_sm/vmcs_sm.c
+++ b/drivers/char/broadcom/vc_sm/vmcs_sm.c
@@ -174,7 +174,7 @@ struct SM_PRIV_DATA_T {
 };
 
 /* Global state information. */
-struct SM_STATE_T {
+struct sm_state_t {
 	struct platform_device *pdev;
 	struct sm_instance *sm_handle;	/* Handle for videocore service. */
 	struct dentry *dir_root;   /* Debug fs entries root. */
@@ -206,7 +206,7 @@ struct SM_STATE_T {
 
 /* ---- Private Variables ----------------------------------------------- */
 
-static struct SM_STATE_T *sm_state;
+static struct sm_state_t *sm_state;
 static int sm_inited;
 
 #if 0
@@ -459,7 +459,7 @@ static unsigned int vmcs_sm_usr_address_from_pid_and_usr_handle(unsigned int
 }
 
 /* Adds a resource mapping to the global data list. */
-static void vmcs_sm_add_map(struct SM_STATE_T *state,
+static void vmcs_sm_add_map(struct sm_state_t *state,
 			    struct sm_resource_t *resource, struct sm_mmap *map)
 {
 	mutex_lock(&(state->map_lock));
@@ -479,7 +479,7 @@ static void vmcs_sm_add_map(struct SM_STATE_T *state,
 }
 
 /* Removes a resource mapping from the global data list. */
-static void vmcs_sm_remove_map(struct SM_STATE_T *state,
+static void vmcs_sm_remove_map(struct sm_state_t *state,
 			       struct sm_resource_t *resource,
 			       struct sm_mmap *map)
 {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1685_35360f9aeff361126cfac5b398a5a1ec040bb32d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1685_35360f9aeff361126cfac5b398a5a1ec040bb32d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
